R/serial.R

```diff
@@ -144,7 +144,7 @@ search_expr_bindings <- function(syms, expr, env) {
 }
 search_symbol_bindings <- function(syms, expr, env) {
   nm <- as_string(expr)
-  syms[[nm]] <- env_type(binding_env(nm, env))
```

Comment:
Could you wrap in `{}` and place the branch on a newline please, e.g.

```r
  if (!nchar(nm)) {
    syms[[nm]] <- env_type(binding_env(nm, env))
  }
```

Also could you add a test for that case?